assets/src/blocks/Covers/CampaignCovers.js

```diff
@@ -0,0 +1,50 @@
+const { __ } = wp.i18n;
+
+export const CampaignCovers = ({ covers, initialRowsLimit, row, loadMoreCovers }) => {
+  const amountPerRow = 3;
+  const showLoadMore = !!initialRowsLimit && covers.length > amountPerRow * initialRowsLimit;
+
+  return (
+    <div className='container'>
+      <div className='thumbnail-largeview-container'>
+        {covers.map((cover, index) => {
+          const { href, image, alt_text, name, src_set, sizes } = cover;
+          const hideCover = !!initialRowsLimit && index >= row * amountPerRow;
+          return (
+            <div key={name} className={`campaign-card-column ${hideCover ? 'hidden' : ''}`}>
```

Comment:
You could use `classnames` here. We're not using it on the front end yet, but it's really small so no significant effect on our bundle size.
```
classNames('campaign-card-column', {hidden:hideCover})